cmd/soroban-rpc/internal/preflight/preflight.go

```diff
@@ -46,7 +46,8 @@ func SnapshotSourceGet(handle C.uintptr_t, cLedgerKey C.xdr_t) C.xdr_t {
 	if err := xdr.SafeUnmarshal(ledgerKeyXDR, &ledgerKey); err != nil {
 		panic(err)
 	}
-	present, entry, err := db.GetLedgerEntry(h.readTx, ledgerKey)
+	// TODO : the expiration sequence here is being ignored for now; it should be passed downstream.
```

Comment:
if this is out of scope for https://github.com/stellar/soroban-tools/issues/1010, should a new ticket be created to capture this gap of downstream use case of LE expiration seq for tracking and also to reference here, i.e. it can help remove the TODO later?

Reply:
yes, I think that it makes sense to create another ticket. As is, it's unblocking the js/cli, so it should good to go.
This comments only suggest that the simulateTransaction performance could improve further here.

Reply:
created issue : https://github.com/stellar/soroban-tools/issues/1025